src/odemis/gui/comp/milling.py

```
@@ -119,6 +122,10 @@ def _add_value_ctrl(self, val, conf):
         if isinstance(val, model.FloatContinuous):
             value_ctrl = UnitFloatCtrl(self._panel, value=val.value,
                                         style=wx.NO_BORDER, **conf)
+            value_ctrl.Bind(wx.EVT_KILL_FOCUS,
+                lambda evt, ctrl=value_ctrl: ctrl.GetEventHandler().ProcessEvent(
```

Ruff (0.14.8)

Unused lambda argument: `evt`

(ARG005)